src/screens/OtherProfile/index.js

```diff
@@ -208,7 +210,7 @@ const OtherProfile = () => {
         anon_user_info_color_code
       };
       const response = await sendAnonymousDMOtherProfile(anonDMParams);
-      await saveChatFromOtherProfile(response, 'sent', true);
+      await saveChatFromOtherProfile(response, 'sent', true, ANON_PM);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a new parameter `ANON_PM` to the function `saveChatFromOtherProfile`. Ensure that this function can handle this new parameter correctly and that it doesn't break existing functionality.
```diff
- await saveChatFromOtherProfile(response, 'sent', true);
+ await saveChatFromOtherProfile(response, 'sent', true, ANON_PM);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->